vertx-web/src/main/java/io/vertx/ext/web/handler/impl/SessionHandlerImpl.java

```diff
@@ -50,6 +51,7 @@ public SessionHandlerImpl(String sessionCookieName, long sessionTimeout, boolean
     this.sessionCookieSecure = sessionCookieSecure;
     this.sessionCookieHttpOnly = sessionCookieHttpOnly;
     this.minLength = minLength;
+    this.sessionCookieMaxAge = 0;
```

Comment:
I'd prefer to see `-1` to denote a non valid max age, since for example `0` can be seen as a `expire now` instead of a marker for non valid max age.